viewer/packages/cad-model/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "main": "index.ts",
   "scripts": {
-    "start": "yarn ws:start",
+    "start": "yarn ws:serve --env testFixture=CadClipping.VisualTest.ts",

```

Comment:
Is this command that would trigger visual test in respective packages? If so, rendering package does not have the same command, how is the visual test for rendering package trigger?